src/modules/fortune-admin/controllers.js

```diff
@@ -2,6 +2,35 @@
 angular.module('fortuneAdmin.Controllers', [
     'fortuneAdmin.Services'
   ])
+  .controller('faActionsCtrl', [
+    '$scope', 
+    'faActionsService',
+    '$sce',
+    '$modal',
+    function ($scope, faActionsService, $sce, $modal) {
+      $scope.actions = faActionsService.getActions('users');
+      $scope.selected = $scope.actions[1];
+      $scope.isModalVisible = false;
+  
+      $scope.selectAction = function (action) {
+        $scope.isModalVisible = action.type === 'modal';
+        console.log($scope.entity);
+        if($scope.isModalVisible && action.createTpl) {
+           $modal.open({
+             template : $sce.trustAsHtml(action.createTpl($scope.entity)),
```

Comment:
Hm, not sure if it's required. If showing html from a variable, the content of the variable must be marked as trusted html, otherwise Angular doesn't like it (for example $scope.content = "<div>Hi</div>; and in the template <div>{{content}}</div>" - will not work, $content must be trusted as html). I needed it initially (wasn't using modal), after which I must've copy pasted...